pkg/apis/upgrade.cattle.io/v1/types.go

```diff
@@ -82,6 +82,8 @@ type PlanSpec struct {
 	ImagePullSecrets []corev1.LocalObjectReference `json:"imagePullSecrets,omitempty"`
 	// Time after a Job for one Node is complete before a new Job will be created for the next Node.
 	PostCompleteDelay *metav1.Duration `json:"postCompleteDelay,omitempty"`
+	// Priority Class Name of Job, if specified.
+	PriorityClassName *string `json:"priorityClassName,omitempty"`
```

Comment:
Question: why `*string` instead of just `string`? The field in pod spec is just a string. Making this a string would simplify both the generated deepcopy and the assignment in pod spec.